src/Microsoft.DotNet.Watcher.Tools/Program.cs

```diff
@@ -6,17 +6,31 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.DotNet.Watcher.Core;
-using Microsoft.Extensions.CommandLineUtils;
 using Microsoft.Extensions.Logging;
 
 namespace Microsoft.DotNet.Watcher.Tools
 {
     public class Program
     {
         private readonly ILoggerFactory _loggerFactory;
+        private readonly CancellationToken _cancellationToken;
+        private readonly TextWriter _out;
 
-        public Program()
+        public Program(CancellationToken cancellationToken)
```

Comment:
Is this ctor needed at all? Why not make the other one public and remove this one? We don't seem to use it (unless there are some tests using it)..